### PR TITLE
[styles] Added login/signup comp

### PR DESCRIPTION
**Pull request by rishichourasia**
Added color variables too

### CODE REVIEW

File: stylesheets/login.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	box-shadow: var(--dark-box-shadow);
+	display: flex;
+	flex-direction: column;
+}
+
+.login-container input {
+	width: auto;
+	border-radius: 4px;
+	font-size: 0.8rem;
+	font-weight: 500;
```
Comment (AnuragSahu11) [inline stylesheets/login.css:17]: You can declare variables for font weight and margin etc too

**Review by AnuragSahu11 — APPROVED** [review]
Code looks good you can merge

File: stylesheets/login.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
+.login-container {
+	width: 370px;
```
Comment (harshitpaliwal95) [inline stylesheets/login.css]: use rem as units

**Review by harshitpaliwal95 — COMMENTED** [review]
- ui looks good

File: stylesheets/login.css (hunk 2 above)
Comment (rishichourasia) [reply stylesheets/login.css]: alright harshit ! thanks , will update that in next commit

File: stylesheets/login.css (hunk 1 above)
Comment (rishichourasia) [reply stylesheets/login.css:17]: ok , noted

**Pull request merged**